src/views/index.js

```diff
@@ -1,5 +1,11 @@
+// IIFE - immediately invoked function expression
 window.onload = function () {
     const appEl = document.querySelector('#app');
-    appEl.innerHTML = LoginPage();
-    AddEventsLogin();
-}
\ No newline at end of file
+    const isLogged = !!sessionStorage.userLogged;
+    if (!isLogged) {
+        interview.navigate("login");
```

Comment:
Since L6 and L9 are identical, aside from the page identifier that gets passed in to `interview.navigate`, it is common practice to keep things nice and DRY via the ternary operator: `interview.navigate(isLogged ? 'evaluations' : 'login');`

Reply:
uuu nice, thank you!